src/locales/cn/translation.json

```diff
@@ -75,6 +75,7 @@
   },
   "contributionBar": {
     "header": "筹款总额",
+    "giftAMeal": "捐爱心餐",
```

Comment:
i just copied this from another gift a meal translation. not sure if it's 100% correct though

Reply:
This looks correct to me!